pkg/container/kubernetes/client.go

```diff
@@ -518,6 +520,18 @@ func (*Client) PullImage(_ context.Context, imageName string) error {
 	return nil
 }
 
+// VerifyImage verifies a container image
+func (*Client) VerifyImage(_ context.Context, serverInfo *registry.Server, imageRef string) (bool, error) {
+	// Create a new verifier
+	v, err := verifier.New(serverInfo)
+	if err != nil {
+		return false, err
+	}
+
+	// Verify the image passing the server info
+	return v.VerifyServer(imageRef, serverInfo)
```

Comment:
Have you tried this in kubernetes? Maybe we should defer this functionality for later. That is, until we figure out how the whole flow would work in k8s.

Reply:
I was about to 😃 The code is using `thv run` so code-wise _should_ be okay, but I'll be able to tell in a bit